modules/platform/nuxeo-elasticsearch/nuxeo-elasticsearch-http-read-only/src/test/java/org/nuxeo/elasticsearch/http/readonly/TestRequestValidator.java

```diff
@@ -47,14 +47,18 @@ public void initValidator() {
     }
 
     @Test
-    public void testCheckValidDocumentId() throws Exception {
+    public void testCheckValidDocumentId() {
         validator.checkValidDocumentId("123");
         exception.expect(IllegalArgumentException.class);
         validator.checkValidDocumentId(null);
     }
 
+    /**
+     * @since 11.4
```

Comment:
why?

Reply:
My bad, it should be a @deprecated since 11.4.